auto_round/mllm/autoround_mllm.py

```diff
@@ -186,18 +187,32 @@ def calib(self, nsamples, bs):
         """
         if isinstance(self.dataset, str):
             dataset = self.dataset.replace(" ", "")
-            self.dataloader = get_mllm_dataloader(
-                template=self.template,
-                model=self.model,
-                tokenizer=self.tokenizer,
-                image_processor=self.image_processor,
-                dataset=dataset, 
-                extra_data_dir=self.extra_data_dir,
-                seqlen=self.seqlen, 
-                bs=bs,
-                seed=self.seed,
-                truncation=self.truncation,
-                )
+            if "pile" in dataset:
```

Comment:
not a good code style,  a slighter better way is first check whether in mllm dataset,  if not then find it in text dataset in llm dataset